src/main/java/com/aws/greengrass/clientdevices/auth/metrics/events/CertificateSubscriptionEvent.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.metrics.events;
+
+import com.aws.greengrass.clientdevices.auth.api.DomainEvent;
+import com.aws.greengrass.clientdevices.auth.api.GetCertificateRequestOptions;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+public class CertificateSubscriptionEvent implements DomainEvent {
+    @Getter
+    private GetCertificateRequestOptions.CertificateType certificateType;
+    @Getter
+    private boolean successfulSubscription;
```

Comment:
prefer enum over boolean. We may have additional states here, like _why_ it failed (ie. network error, permissions error).